src/main/kotlin/com/sourcegraph/cody/sidebar/WebThemeController.kt

```diff
@@ -14,19 +18,30 @@ class WebThemeController {
         invokeLater { themeChangeListener?.invoke(getTheme()) }
       }
     }
+
+    ApplicationManager.getApplication()
+        .messageBus
+        .connect(parentDisposable)
+        .subscribe(
+            UISettingsListener.TOPIC,
+            UISettingsListener { _ -> invokeLater { themeChangeListener?.invoke(getTheme()) } })
   }
 
   fun setThemeChangeListener(listener: (WebTheme) -> Unit) {
     themeChangeListener = listener
   }
 
   fun getTheme(): WebTheme {
-    val themeVariables =
+    val colorVariables =
         UIManager.getDefaults()
             .filterValues { it is Color }
             .mapKeys { toCSSVariableName(it.key.toString()) }
             .mapValues { toCSSColor(it.value as Color) }
-    return WebTheme(ThemeUtil.isDarkTheme(), themeVariables)
+
+    val fontSizeVariable =
+        (toCSSVariableName("font-size") to "${UISettings.getInstance().fontSize}px")
+
+    return WebTheme(ThemeUtil.isDarkTheme(), colorVariables + fontSizeVariable)
```

Comment:
What are these variables? Strings of css properties?

Reply:
They are used for css substitution here:
https://github.com/sourcegraph/cody/pull/6134/files/d03e45208f6030dc3064c77ec49552caa20b9f60#diff-8765a30f3f70a874222d56f0f4eb9c7380b324d07902ae0227afab7ce7afd4d7